src/components/aktivitetskrav/vurdering/OppfyltAktivitetskravSkjema.tsx

```diff
@@ -4,17 +4,21 @@ import {
   OppfyltVurderingArsak,
 } from "@/data/aktivitetskrav/aktivitetskravTypes";
 import { oppfyltVurderingArsakTexts } from "@/data/aktivitetskrav/aktivitetskravTexts";
-import { VurderAktivitetskravArsakRadioGruppe } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravArsakRadioGruppe";
+import {
+  vurderAktivitetskravArsakFieldName,
+  VurderAktivitetskravArsakRadioGruppe,
+} from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravArsakRadioGruppe";
 import { VurderAktivitetskravSkjema } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravSkjema";
 import { useAktivitetskravVurderingSkjema } from "@/hooks/aktivitetskrav/useAktivitetskravVurderingSkjema";
+import { vurderAktivitetskravBeskrivelseFieldName } from "@/components/aktivitetskrav/vurdering/VurderAktivitetskravBeskrivelse";
 
 const texts = {
   title: "Aktivitetskravet er oppfylt",
 };
 
 interface OppfyltAktivitetskravSkjemaValues {
-  beskrivelse: string;
-  arsak: OppfyltVurderingArsak;
+  [vurderAktivitetskravBeskrivelseFieldName]: string;
+  [vurderAktivitetskravArsakFieldName]: OppfyltVurderingArsak;
```

Comment:
Hvordan funker slike felter, egentlig?
Ser at `vurderAktivitetskravBeskrivelseFieldName = "beskrivelse"` et annet sted i appen.

Reply:
Tanken var å ha navnet på feltet definert ett sted - i komponenten for feltet (`<VurderAktivitetskravBeskrivelse>`).
Og så bruker man den i skjemaene som skal ha feltet, så `OppfyltAktivitetskravSkjemaValues` vil f.eks. bli: 
`{ beskrivelse: string, arsak: OppfyltVurderingArsak }`